lntest/itest/assertions.go

```diff
@@ -399,8 +407,12 @@ func findWaitingCloseChannel(pendingChanResp *lnrpc.PendingChannelsResponse,
 
 // waitForChannelPendingForceClose waits for the node to report that the
 // channel is pending force close, and that the UTXO nursery is aware of it.
-func waitForChannelPendingForceClose(ctx context.Context,
-	node *lntest.HarnessNode, fundingChanPoint *lnrpc.ChannelPoint) error {
+func waitForChannelPendingForceClose(node *lntest.HarnessNode,
+	fundingChanPoint *lnrpc.ChannelPoint) error {
+
+	ctxb := context.Background()
+	ctx, cancel := context.WithTimeout(ctxb, channelCloseTimeout)
```

Comment:
Since we're only going to wait for `defaultTimeout` below anyway, this could also just be the default timeout.

Reply:
Changed!